### PR TITLE
News editor appears for a few seconds before appearance of message Permission denied

### PR DESCRIPTION
**Pull request by lbengayed**

### CODE REVIEW

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 1]
```diff
@@ -200,7 +200,8 @@ export default {
       attachmentsChanged: false,
       imagesURLs: new Map(),
       uploading: false,
-      canCreatNews: true,
+      canCreatNews: '',
```
Comment (ahamdi) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue:203]: Won't it be sufficient to initialize canCreateNews to false ? without adding the other variable canNotCreatNews ?

Comment (lbengayed) [reply webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue:203]: I do it, but I found in the other case, the message 'Permission denied' appears for a few seconds before appearance of News editor if i change canCreateNews to false

Comment (ahamdi) [reply webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue:203]: I recommend to use a loading component before initiating the DOM instead .

Comment (lbengayed) [reply webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: done!

**Pull request merged**